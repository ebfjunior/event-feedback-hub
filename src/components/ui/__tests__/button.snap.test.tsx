import React from 'react';
import { render } from '@testing-library/react';
import { Button } from '../button';

describe('ui/Button (snapshot)', () => {
	it('renders variants', () => {
		const { container } = render(
			<div>
				<Button>Default</Button>
				<Button variant="secondary">Secondary</Button>
				<Button size="sm">Small</Button>
			</div>,
		);
		expect(container.firstChild).toMatchSnapshot();
	});
});


