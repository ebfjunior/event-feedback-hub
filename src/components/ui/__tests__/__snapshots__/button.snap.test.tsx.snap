// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ui/Button (snapshot) > renders variants 1`] = `
<div>
  <button
    class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:opacity-50 bg-primary text-primary-foreground shadow hover:bg-primary/90 h-9 px-4 py-2"
  >
    Default
  </button>
  <button
    class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:opacity-50 bg-secondary text-secondary-foreground shadow-sm hover:bg-secondary/80 h-9 px-4 py-2"
  >
    Secondary
  </button>
  <button
    class="inline-flex items-center justify-center whitespace-nowrap font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:opacity-50 bg-primary text-primary-foreground shadow hover:bg-primary/90 h-8 rounded-md px-3 text-xs"
  >
    Small
  </button>
</div>
`;
